### ISSUE TITLE
Accessibility violation in empty tables

### ISSUE THREAD

**Issue by caritaou** [elyra-ai/canvas#2501]
ref 4050

![Image](https://github.com/user-attachments/assets/db198a73-a62b-4ad6-afef-0b33402a36c6)
